sktime/forecasting/trend/_spline_trend_forecaster.py

```diff
@@ -0,0 +1,291 @@
+# !/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements SplineTrendForecaster."""
+
+__author__ = ["jgyasu", "tensorflow-as-tf", "mloning", "aiwalter", "fkiraly"]
+__all__ = ["SplineTrendForecaster"]
+
+import numpy as np
+import pandas as pd
+from sklearn.base import clone
+from sklearn.linear_model import LinearRegression
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import SplineTransformer
+
+from sktime.forecasting.trend import PolynomialTrendForecaster
+from sktime.forecasting.trend._util import _get_X_numpy_int_from_pandas
+
+
+class SplineTrendForecaster(PolynomialTrendForecaster):
+    """
```

Comment:
I think it's preferred to start docstring on same line as initial triple quotes. See this: https://numpydoc.readthedocs.io/en/latest/validation.html#built-in-validation-checks